### PR TITLE
feat: allow to pass `<sectionname>` as first argument

### PR DESCRIPTION
**Pull request by drahnr**

### CODE REVIEW

File: cli.rs [hunk 1]
```diff
@@ -6,37 +6,45 @@ use libsui::PortableExecutable;
 use libsui::utils;
 
 static TEST_ICO: &[u8] = include_bytes!("./tests/test.ico");
-const HELP: &str = r#"Usage: sui <exe> <data_file> <output>"#;
+const HELP: &str = r#"Usage: sui <sectionname> <exe> <data_file> <output>"#;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    if let Some(section) = find_section("__SUI") {
+	let args: Vec<String> = std::env::args().collect();
+	let sectionname = args.len() <1 {
```
Comment (littledivy) [inline cli.rs]:
```suggestion
	let sectionname = args.len() < 1 {
```

**Review by littledivy — COMMENTED** [review]
Can you run `cargo fmt` to format the code? Thanks for the PR

**Comment by drahnr** [comment]
Done.

Would you be interested in extending the CLI slightly using `clap` and avoiding to read the extra data to memory?

**Comment by littledivy** [comment]
> Would you be interested in extending the CLI slightly using clap and avoiding to read the extra data to memory?

Sure! Feel free to open a PR/issue.

**Pull request merged**